Formula/steem.rb

```diff
@@ -0,0 +1,33 @@
+class Steem < Formula
+  desc "Proof of Work blockchain with an unproven consensus algorithm."
+  homepage "https://steemit.com/"
+  url "https://github.com/steemit/steem.git",
+      :tag => "v0.8.5b",
```

Comment:
I don't have strong feelings either way, honestly. Happy for you to overrule if you prefer.